### DISCUSSION TITLE
timestamps for coverage results

### DISCUSSION THREAD

**Discussion by safwatalikhan** [pilgun/acvtool#47] [Q&A]
Hi, Is there a way to get timestamps for the coverage results? Ideally showing which instruction/method/class were executed during which time.

**Comment by pilgun** [comment]
It’s a quite a bit challenging to implements at the current stage, but
possible. It will also affect the performance of the app.

**Reply by safwatalikhan** [reply]
thanks for replying. Can you let me know how would I go about implementing it?
